doc/configuration.md

```diff
@@ -97,3 +97,17 @@ This flag periodically prunes the consumed tx_out table. So it allows to query f
 without having to maintain the whole tx_out table. Deletes to `tx_out` are propagated to `ma_tx_out`
 through foreign keys. If this is set once, then it must be always set on following executions of
 db-sync. Failure to do this can result in crashed and db-sync currently has no way to detect it.
+
+### --bootstrap
+
+This flag results in a similar db schema as using `--prune-tx-out`, except it syncs faster. The difference is that instead of inserting/updating/deleting outputs, it delays the intestion of
```

Comment:
Is this a typo (intestion -> ingestion)?

Reply:
Indeed meant insertion